crates/nostr-sqldb/migrations/sqlite/2025-04-11-095120_events/up.sql

```diff
@@ -0,0 +1,27 @@
+-- The actual event data
+CREATE TABLE events (
+    id VARCHAR(64) PRIMARY KEY,
+    pubkey VARCHAR(64) NOT NULL,
+    created_at BIGINT NOT NULL,
+    kind BIGINT NOT NULL,
+    payload BLOB NOT NULL,
```

Comment:
If possible I think is better to remove the `payload`, to avoid saving events twice. Are you using it for any specific reason?

Reply:
Yes the payload is the actual event data, everything else is just indexes and state.